wurst/objects/abilities/beastMaster/Pets/PetTaming.wurst

```diff
@@ -15,41 +15,53 @@ import Pets
 import StringExtensions
 import ToolTipsUtils
 import PlayerExtensions
+import LinkedList
 
 let PET_MAX_MANA = 6
 
+let TAMEABLE_PETS = asList(
+    " Elk,",
+    " Wolf,",
+    " Bear"
+)
+
+let PETS_TT = map((animal, _) -> animal.color(COLOR_YELLOW), TAMEABLE_PETS).join()
```

Comment:
This is better but you're doing the work of `join` manually. You can use `map((animal, _) -> animal.color(COLOR_YELLOW), asList("Elk", "Wolf", "Bear").join(", ")` to fill in the separator yourself. You'll then need to add spacing around `PETS_TT` below, or just use `.format()` to inline it.